### DISCUSSION TITLE
Wayland Causing Distrobox to  Crash system

### DISCUSSION THREAD

**Discussion by xensurve** [89luca89/distrobox#1241] [General]
OS : ARCH
DESKTOP : KDE 5.27
DISTROBOX : 1.6.0-1

When ever I log in with wayland session It freezes the whole system I cant move the mouse it just locks up

Anyone else having this problem. ?

**Comment by traverseda** [comment]
I get this problem when I have a distrobox container running and resume from suspend on nixos
